New bios 2023

saisha-r opened the pull request.

saisha-r committed:
add cory

saisha-r committed:
change some members to alum

saisha-r committed:
Update citations

saisha-r committed:
add more members

saisha-r committed:
update laura and saisha titles

saisha-r committed:
update cory picture

saisha-r committed:
update cory bio

saisha-r committed:
move brynn to staff

saisha-r committed:
change description on brynn to RA

saisha-r committed:
update brynn bio

saisha-r committed:
Update laura-elliott.md

saisha-r committed:
Update citations

saisha-r committed:
Update max-mascini.md

aaronjnewman commented:
Looks good! Although Brynn looks totally high in her pic :laughing:

Cory can actually be moved to alumni bc his project is officially completed. Likewise, Lyam is officially no longer a NCIL student as of July 1 so he's now alumni.

We should also start collecting info on incoming students (at least those you can, like Clare) and adding them as we can. There are a number of even current ones missing from the page now, like Liam P., Kya, Rafael (if we're including Mayank we should include all Colin's students). I'd prefer to have placeholders with names, even if we don't have a pic and bio, so that the website reflects the actual size of the lab (which is helpful when I tell inquiring students we're full!).

thanks

saisha-r committed:
add liam pickles

saisha-r committed:
move cory and lyam to alum

saisha-r committed:
Update liam-pickles.md

saisha-r committed:
add sarah haggett

saisha-r committed:
add rafiah mir

saisha-r committed:
add emily

saisha-r committed:
add images, update roles, etc.

saisha-r committed:
add people

saisha-r committed:
add paige

saisha-r committed:
add jana email

saisha-r committed:
move mayank to alum

saisha-r committed:
fix typo

saisha-r commented:
@aaronjnewman take a look and let me know what you think! There might be some people I am missing or people who need to be moved in/out of alum

saisha-r committed:
add eloise

saisha-r committed:
add eloise's email

saisha-r committed:
move brynn and eloise to alum

saisha-r committed:
Update citations

saisha-r committed:
update laura earl

saisha-r committed:
Merge branch 'add-bios-2023' of https://github.com/NeuroCognitiveImagingLab/NCIL-website into add-bios-2023

saisha-r committed:
move hamid to alum

saisha-r committed:
update undergrad photos

saisha-r committed:
fix photos and add clare

saisha-r committed:
fix alum

saisha-r committed:
fix clare's pic and remove postdoc

saisha-r commented:
I went through the current lab members pictures and zoomed in on the one's that needed it! If you would like for me to do this with the alum too let me know. I also might need to ask Danny for a new pic since his zoomed up looks quite grainy. Take a look and let me know your thoughts! @aaronjnewman

saisha-r commented:
Also, what section should Nana go under since she is a volunteer?

aaronjnewman commented:
Put nana under undergrads


Aaron J Newman, PhD (he/him)

Professor
Chair, Department of Psychology & Neuroscience<https://www.dal.ca/faculty/science/psychology_neuroscience.html>
Director, NeuroCognitive Imaging Lab (NCIL)<http://ncil.science/>
Director, SURGE Science Innovation Sandbox<http://surgeinnovation.ca/>
Author, Research Methods for Cognitive Neuroscience<https://us.sagepub.com/en-us/nam/research-methods-for-cognitive-neuroscience/book242924>
Editor, Changing Brains: Essays on Neuroplasticity in Honor of Helen J. Neville<https://www.routledge.com/Changing-Brains-Essays-on-Neuroplasticity-in-Honor-of-Helen-J-Neville/Newman-Grossi/p/book/9780367358693>

FACULTY OF SCIENCE | Department of Psychology & Neuroscience
FACULTY OF MEDICINE | Departments of Pediatrics, Psychiatry, and Surgery
Life Sciences Centre, Room P3333
+1.902.488.1973 | ***@***.***

DALHOUSIE UNIVERSITY
dal.ca

Dalhousie University is located in Mi'kma'ki, the ancestral and unceded territory of the Mikmaq. We are all Treaty people.

We recognize that African Nova Scotians are a distinct people whose histories, legacies and contributions have enriched that part of Mi'kma'ki known as Nova Scotia for over 400 years.

Want to book a meeting? Click here<https://fantastical.app/aaronnewman-H23O/meet>

________________________________
From: Saisha Rankaduwa ***@***.***>
Sent: Monday, October 21, 2024 3:22:06 PM
To: NeuroCognitiveImagingLab/NCIL-website ***@***.***>
Cc: Aaron Newman ***@***.***>; Mention ***@***.***>
Subject: Re: [NeuroCognitiveImagingLab/NCIL-website] New bios 2023 (PR #40)

CAUTION: The Sender of this email is not from within Dalhousie.

Also, what section should Nana go under since she is a volunteer?

—
Reply to this email directly, view it on GitHub<https://github.com/NeuroCognitiveImagingLab/NCIL-website/pull/40#issuecomment-2427417159>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/APCKA5QRHEDEZ7GI3WLDJHTZ4VA45AVCNFSM6AAAAABA4F2KIKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMRXGQYTOMJVHE>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

saisha-r committed:
add Nana, edit Jana

saisha-r committed:
write Research Assistant instead of RA

saisha-r commented:
I added Nana, and adjusted her and Jana's role description RA trainee (Cindy's suggestion). Some images of students are missing because I wasn't sent any. Also, I was wondering if Rafiah and Mobina should be moved to alum.

@aaronjnewman

aaronjnewman committed:
update gitignore to help resolve conflicts

aaronjnewman committed:
Update citations

aaronjnewman committed:
fix conflicts?
